packages/api/src/platforms/vtex/clients/fetch.ts

```diff
@@ -25,7 +26,14 @@ export const fetchAPI = async (
       options.storeCookies(response.headers)
     }
 
-    return response.status !== 204 ? response.json() : undefined
+    const json = response.status !== 204 ? await response.json() : undefined
+
+    if (typeof json === 'string' && json.toLowerCase() === 'invalid user') {
```

Comment:
We will have other private routes soon.  `json.toLowerCase() === 'invalid user'` is specific return of `reviews-and-rating api`, right? I think this fetch method should be generic.